src/scenes/QtumPrediction/index.js

```diff
@@ -4,42 +4,22 @@ import { inject, observer } from 'mobx-react';
 import InfiniteScroll from '../../components/InfiniteScroll';
 import theme from '../../config/theme';
 import Loading from '../../components/EventListLoading';
-import EventCard from './components/EventCard';
+import EventCard from '../../components/EventCard';
 import TopActions from './components/TopActions';
 
 
 @inject('store')
 @observer
 export default class QtumPrediction extends Component {
-  counterInterval = undefined;
-
-  constructor() {
-    super();
-    this.state = {
-      increasingCount: 0,
-    };
-  }
-
   componentDidMount() {
-    this.props.store.qtumPrediction.init();
-    const interval = 1;
-    this.counterInterval = setInterval(() => {
-      const { increasingCount } = this.state;
-      this.setState({
-        increasingCount: increasingCount + (interval / 1000000000),
-      });
-    }, interval * 1000);
-  }
-
-  componentWillUnmount() {
-    clearInterval(this.counterInterval);
+    this.props.store.ui.initIncreasingCount(1);
```

Comment:
I think we still need to reset the counter param to 0 at every refresh of the page

Reply:
I am not quite sure why this method of figuring out the time left was used. This logic seems like it could be optimized:
- app init
- globalStore: constructor -> setInterval()
- var currentTime = moment.unix()
- interval increments the currentTime by 1 every second
- EventCards -> endTimeUnix - currentTimeUnix = timeLeftUnix

What do you think?

Reply:
Yes it looks better. I will do this way.